data/campaigns/Heir_To_The_Throne/achievements.cfg

```diff
@@ -182,6 +182,75 @@
         hidden_hint= _ "Look around a bit for a hidden tunnel"
     [/achievement]
 
+    [achievement]
+        id="HttT_S15_Shields_of_Knalga"
+        name= _ "Shields of Knalga"
+        name_completed= _ "Shields of Knalga"
+        description= _ "Save Lord Geldar in <i>The Lost General</i>"
+        description_completed= _ "Acquired the support of the <i>Dwarven Guardsman</i>"
+        icon="units/dwarves/guard-defend-2.png~RC(magenta>red)~BLIT(misc/achievement-frame-gold.png~SCALE(72,72), 0, 0)"
+        icon_completed="units/dwarves/guard-defend-2.png~RC(magenta>red)~BLIT(misc/achievement-frame-gold.png~SCALE(72,72), 0, 0)"
+        hidden=yes
+        hidden_name=_"???"
+        hidden_hint= _ "Be a hero"
+    [/achievement]
+
+    [achievement]
+        id="HttT_S15_Into_the_Maelstorm"
+        name= _ "Into the Maelstorm"
+        name_completed= _ "Into the Maelstorm"
```

Comment:
Um… I guess this isn't really your fault in this case but… isn't the correct spelling "Maelstrom"?